manifests/charts/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -37,6 +46,15 @@ spec:
         host: istiod.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
         port:
           number: 15012
+  {{- if  .Values.global.centralIstiod }}
```

Comment:
Did you test this with any other https service running on 443 ? I can't see how it would work.

Reply:
I did a bit more testing, and found out calling the cross cluster service isn't working, for example:

sleep from main cluster calls httpbin on remote cluster is broken (main cluster has httpbin service but no deployment)

```
/ # curl httpbin:8000/headers
upstream connect error or disconnect/reset before headers. reset reason: connection failure/ # 
```

If I move the webhook to a different port for the gw/vs/dr than 443, then the above test works fine.

cc @howardjohn 

I think this means we will have to use a different port for webhook.  I would not be surprised if any https service running on 443 is impacted if we use the same port as well.

Reply:
Well - you can use port 443 'properly' - with secret-based certificates that Ingress gateway will
load and terminate HTTPs connection, then forward to istiod.

We use the 'pass-through-tcp' for 15012 because Istiod is supposed to use mTLS authentication - I don't think that's an issue for webhooks.

Reply:
I see, that can be a todo for the future to investigate terminating https connection at the ingress gw for webhook.

Reply:
I am working on getting this set up, I think its pretty close to working. Will share soon

Reply:
awesome, pls do share!